meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -61,6 +61,7 @@ FILES:${PN}-get = "${bindir}/${PN}-get"
 FILES:${PN}-lite = " \
                     ${bindir}/${PN}-lite \
                     ${nonarch_libdir}/tmpfiles.d/${PN}-lite.conf \
+                    ${bindir}/fioapp \
```

Comment:
i'm guessing the name of this is going to change once aklite is updated. I guess the sha to aklite will also have to get bumped here.

Reply:
Correct. I just wanted to check whether it works from end2end perspective as well as get some feedback before finalizing this work.